wordpress-mobile/WordPress-Android
# Issue/13326 fix threat list item UI issue

This PR fixes a threat list item UI issue reported in https://github.com/wordpress-mobile/WordPress-Android/pull/13994#issuecomment-775814436).

To test:
- Open Scan on a site with scan capability and with fixed or ignored threats (e.g. [pressable-jetpack-daily-scan](https://pressable-jetpack-daily-scan.mystagingwebsite.com/)))
- Click History menu, wait for history records to be displayed
- Lock your phone, wait for 5'' secs, and unlock it
- Notice that the threat list item is rendered properly

PR submission checklist:
